### PR TITLE
add Pushsafer notification service

### PR DESCRIPTION
**Pull request by appzer**

### CODE REVIEW

**Comment by dgw** [comment]
I'm seeing whitespace inconsistencies in some added lines.

**Comment by appzer** [comment]
i formated the code

File: README.md [hunk 1]
```diff
@@ -244,6 +245,7 @@ to something similar to "http://domain/#channel/2011-03-09 14:25:09", or
     *   `boxcar`
     *   `nma`
     *   `pushover`
+	*   `pushsafer`
```
Comment (dgw) [inline README.md]: Tab character here

File: README.md [hunk 2]
```diff
@@ -284,6 +286,8 @@ to something similar to "http://domain/#channel/2011-03-09 14:25:09", or
     number. The number must be in international format.
 
     When using Telegram, this is the id of the chat that receives the message.
+	
+	When using Pushsafer, this is the id or group id of your devices.
```
Comment (dgw) [inline README.md]: Tab character here

File: README.md [hunk 3]
```diff
@@ -333,9 +337,9 @@ to something similar to "http://domain/#channel/2011-03-09 14:25:09", or
 *   `message_sound` Default: ` `
 
     Notification sound to play with the push notification.
-    Supported under Pushover, Faast, and Boxcar 2.  Must be chosen from the list of
-    [Pushover sounds](https://pushover.net/api#sounds), [Faast sounds](http://developer.faast.io/)
-    or [Boxcar 2 sounds](https://boxcar.uservoice.com/knowledgebase/articles/306788-how-to-send-your-boxcar-account-a-notification).
+    Supported under Pushover, Pushsafer, Faast, and Boxcar 2.  Must be chosen from the list of
+    [Pushover sounds](https://pushover.net/api#sounds), [Pushsafer sounds](https://www.pushsafer.com/pushapi),
+	[Faast sounds](http://developer.faast.io/) or [Boxcar 2 sounds](https://boxcar.uservoice.com/knowledgebase/articles/306788-how-to-send-your-boxcar-account-a-notification).
```
Comment (dgw) [inline README.md]: Tab character here

**Comment by appzer** [comment]
all ok now

**Comment by jreese** [comment]
Looks great!  Thanks for the help in reviewing, @dgw.

**Pull request merged**